library/ssl_misc.h

```diff
@@ -756,11 +756,9 @@ struct mbedtls_ssl_handshake_params {
  * access to ecdh_ctx structure is needed for MBEDTLS_ECDSA_C which does not
  * seem correct.
  */
-#if defined(MBEDTLS_ECDH_C) || defined(MBEDTLS_PK_CAN_ECDSA_SOME)
-#if !defined(MBEDTLS_USE_PSA_CRYPTO)
+#if defined(MBEDTLS_ECDH_C) && !defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
Thanks for fixing this. However, please remove the comment above as it doesn't make sense any more :)

Btw, there's one more occurrence in `ssl.h` around line 47, with a similar comment that we should be able to remove now as well.

Reply:
Yes, of course, sorry for that